app/adapters/newspost.js

```diff
@@ -0,0 +1,15 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+    host: 'https://api.currentsapi.services',
+    namespace: 'v1',
+    apiKey: 'AVf3jCmyyRrC8jF9b7vmlNcnoTJefeC7Gz0c3LyedvLGE_bo',
```

Comment:
@shenbaga-praburaj , this is fine for the sample app for now. But the general basic practice is not to have apiKey in adapter files. These should come from config files, which are not shipped with production code. As API keys shouldn't be exposed and this is a security concern. When you work on our freshteam app just remember this point :)